src/include/stir/recon_buildblock/GibbsPrior.inl

```diff
@@ -0,0 +1,623 @@
+/*
+    Copyright (C) 2025, University College London
+    Copyright (C) 2025, University of Milano-Bicocca 
```

Comment:
a lot of stuff copied from the old QuadraticPrior presumably, so need to preserve its copyright

Reply:
Checking some history, we need to add authors
- Robert Twyman 
- Sanida Mustafovic

copyright: 
- UCL copyright 2022
- Hammersmith Imanet Copyright 2001 - 2005

thanks!